lib/activity_percentage_calculator.rb

```diff
@@ -14,6 +14,7 @@ def outside_activities
   end
 
   def percentage_of_outside_activities
+    return 0 if outside_activities == 0 && inside_activities == 0
```

Comment:
On another day, I would say that I think there are some built in methods that make this kind of math cleaner, but this works so :+1: